src/ontology/mondo-edit.obo

```diff
@@ -520739,6 +520724,32 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6622" xsd:anyURI
 property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/640/congenital-amegakaryocytic-thrombocytopenia" xsd:anyURI {source="GARD:0000640"}
 
+[Term]
+id: MONDO:0800453
+name: juvenile absence epilepsy
+def: "Juvenile absence epilepsy (JAE) is a genetic epilepsy with onset occurring around puberty. JAE is characterized by sporadic occurrence of absence seizures, frequently associated with a long-life prevalence of generalized tonic-clonic seizures (GTCS) and sporadic myoclonic jerks." [Orphanet:1941]
+subset: gard_rare {source="GARD:2162"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:1941"}
+subset: orphanet_rare {source="Orphanet:1941"}
+subset: predisposition
+subset: rare
+synonym: "epilepsy juvenile absence" RELATED [GARD:0002162]
+synonym: "JAE" EXACT ABBREVIATION [Orphanet:1941]
+xref: DOID:0060172 {source="MONDO:equivalentTo"}
+xref: GARD:2162 {source="Orphanet:1941"}
+xref: ICD10CM:G40.3 {source="Orphanet:1941", source="Orphanet:1941/attributed", source="Orphanet:1941/ntbt"}
+xref: MESH:C535495 {source="Orphanet:1941", source="Orphanet:1941/e"}
+xref: NCIT:C129868 {source="MONDO:equivalentTo"}
+xref: Orphanet:1941 {source="MONDO:equivalentTo", source="OMIM:607631"}
+xref: SCTID:230413002 {source="MONDO:equivalentTo"}
+xref: UMLS:C4317339 {source="MONDO:equivalentTo"}
+is_a: MONDO:0020573 ! inherited disease susceptibility
+is_a: MONDO:0100030 {source="https://orcid.org/0000-0001-8486-0558"} ! adolescent/adult-onset epilepsy syndrome
+relationship: has_characteristic MONDO:0021152 {source="Orphanet:1941"} ! inherited
+relationship: predisposes_towards MONDO:0800453 ! juvenile absence epilepsy
```

Comment:
This is incorrect, the juvenile absence epilepsy is not a susceptibility

Reply:
fixed